### ISSUE TITLE
Projects do not load (Nothing but white page)

### ISSUE THREAD

**Issue by jacobamey** [rundeck/rundeck#2549]
Issue type: Bug report

**My Rundeck detail**

* Rundeck version: 2.8.2-1
* install type: rpm
* OS Name/version: CentOS Linux release 7.3.1611 (Core)
* DB Type/version: h2

**Expected Behavior**
Click on project and receive project with Jobs

**Actual Behavior**

Click on the project and am presented with a white page that never loads instead.

**How to reproduce Behavior**
After logging into rundeck, click on a Project, The project will not load but instead go to a white page with no error or information for use.

**Error**
Request: /rundeck/menu/admin
Message: 
Caused by: Error applying layout : base 
Class: gsp_rundeck_layoutsbase_gsp 
At Line: [0] 
Code Snippet(s):

org.codehaus.groovy.grails.web.pages.exceptions.GroovyPagesException: Error applying layout : base
	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:696)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1515)
	at grails.plugin.cache.web.filter.PageFragmentCachingFilter.doFilter(PageFragmentCachingFilter.java:198)
	at grails.plugin.cache.web.filter.AbstractFilter.doFilter(AbstractFilter.java:63)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1486)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1486)
...
Caused by: org.codehaus.groovy.grails.web.pages.exceptions.GroovyPagesException: Error processing GroovyPageView: Error executing tag <g:render>: null
	... 44 more
Caused by: org.codehaus.groovy.grails.web.taglib.exceptions.GrailsTagException: Error executing tag <g:render>: null
	at gsp_rundeck_layoutsbase_gsp$_run_closure2.doCall(gsp_rundeck_layoutsbase_gsp.groovy:178)
	at gsp_rundeck_layoutsbase_gsp.run(gsp_rundeck_layoutsbase_gsp.groovy:189)
	... 44 more
Caused by: java.lang.NullPointerException

**Comment by jacobamey** [comment]
Also, am finding these errors in the service.log

ERROR GrailsExceptionResolver: NullPointerException occurred when processing request: [GET] /rundeck/project/DPC_Automation_and_Deployments/jobs
ERROR GrailsExceptionResolver: NullPointerException occurred when processing request: [GET] /rundeck/project/DPC_Automation_and_Deployments/configure

**Comment by gschueler** [comment]
is there any stacktrace inside the service.log?

**Comment by jacobamey** [comment]
I believe I found the issue.

test=“${auth.resourceAllowedTest(type: ‘resource’, kind: ‘system’,
                    action: [AuthConstants.ACTION_READ, AuthConstants.ACTION_ADMIN], any: true, context: ‘application’)}“>

[12:58] 

It looks to be an issue within the ACL policy set. I removed my standard user ACL and am just running with my admin setup and all is well now. I'll need to run through my system portion of the ACL to see where I goofed.

This all looks like it's on my end so I'll close the ticket out.
